### PR TITLE
Get endpoints and authentications from sources

### PR DESCRIPTION
**Pull request by agrare**
These are going to move to the sources-api

### CODE REVIEW

File: bin/openshift-operations [hunk 1]
```diff
@@ -12,5 +12,10 @@ TopologicalInventoryApiClient.configure do |config|
   config.host = "#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"]}"
 end
 
+SourcesApiClient.configure do |config|
+  config.scheme = "http"
+  config.host   = "#{ENV["SOURCES_HOST"]}:#{ENV["SOURCES_HOST"]}"
```
Comment (carbonin) [inline bin/openshift-operations]: Think this should be `"#{ENV["SOURCES_HOST"]}:#{ENV["SOURCES_PORT"]}"`

File: bin/openshift-operations [hunk 2]
```diff
@@ -12,5 +12,10 @@ TopologicalInventoryApiClient.configure do |config|
   config.host = "#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"]}"
 end
 
+SourcesApiClient.configure do |config|
+  config.scheme = "http"
```
Comment (carbonin) [inline bin/openshift-operations]: This will also be provided as an ENV var, although it's unlikely to change

File: bin/openshift-operations (hunk 1 above)
Comment (agrare) [reply bin/openshift-operations]: :+1: derp, done

File: bin/openshift-operations (hunk 2 above)
Comment (agrare) [reply bin/openshift-operations]: Updated to use `ENV["SOURCES_SCHEME"]`

File: bin/openshift-operations [hunk 3]
```diff
@@ -12,5 +12,10 @@ TopologicalInventoryApiClient.configure do |config|
   config.host = "#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"]}:#{ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"]}"
 end
 
+SourcesApiClient.configure do |config|
+  config.scheme = "#{ENV["SOURCES_SCHEME"]}"
```
Comment (carbonin) [inline bin/openshift-operations]: I think the env value is already a string so the interpolation is unnecessary.

Comment (agrare) [reply bin/openshift-operations]: :+1:

**Pull request merged**